app/src/main/java/com/example/tasky/MainActivity.kt

```diff
@@ -90,9 +77,20 @@ fun NavGraphBuilder.authGraph(navController: NavController, loginViewModel: Logi
     }
 }
 
+fun NavGraphBuilder.calendarGraph(navController: NavController, agendaViewModel: AgendaViewModel) {
+    composable<AgendaNav> {
+        AgendaContent(navController = navController, agendaViewModel = agendaViewModel)
+    }
+}
```

Comment:
Extracting this into a separate function doesn't make it a navigation graph yet 😄 
You'll need this inside:
```
navigation<CalendarGraph>(startDestination = AgendaNav) {
    composable<AgendaNav> {
    
    }
}
```